fr/_posts/2024-06-05-systeme-design-unifie.md

```diff
@@ -0,0 +1,37 @@
+---
+altLangPage: /2024/06/05/unified-design-system.html
+css:
+-  ../../../css/blogue.css
+- https://use.fontawesome.com/releases/v5.15.4/css/all.css
+date: 2024-06-05
+dateModified: 2024-06-05
+description: "Découvrez comment nous créons une source unifiée alliant code, conception et documentation afin d’aider les équipes à créer des sites Web et des applications."
+title: "Système de design unifié et mise à jour sur la fusion du BTN et du SNC"
+---
+
+<h2>Vers un système de design unifié pour le gouvernement du Canada</h2>
+<p>L’une des façons de tendre à l’uniformité de la prestation des services consiste à recourir à un système de design cohésif. Au gouvernement du Canada, nous mettons au point une nouvelle version de système de design qui permettra de répondre aux besoins changeants des équipes de prestation de services et du public à l’ère numérique.</p>
+
+<p><b>En savoir plus&nbsp;:</b></p>
+<div class="col-md-12 mrgn-bttm-xl">
+<div class="col-md-6">
+        <img class="img-responsive thumbnail" src="/images/DTO_CDS_DesignSystem_Blog_Post_FR.jpg" alt=""/>
+        <h3><a class="stretched-link" href="https://numerique.canada.ca/2024/05/27/vers-un-syst%C3%A8me-de-design-unifi%C3%A9-pour-le-gouvernement-du-canada/">Vers un système de design unifié pour le gouvernement du Canada</a></h3>
+</div>
+</div>
+
+
+<div class="clearfix">
```

Comment:
```suggestion
<div class="clearfix">
```
Clearfix are no longer necessary since the grid system doesn't rely on floats as much.